src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -4192,6 +4192,9 @@ void ExceptionTracker::MakeCallbacksRelatedToHandler(
 // Notes:
 //    If the exception is intercepted, this function never returns.
 //
+#ifdef HOST_UNIX
```

Comment:
this definition looks odd here. Can its implementation just be moved up? Also the comment above now is misplaced.